### PR TITLE
Add new translators to markdown document

### PR DESCRIPTION
**Pull request by TomKellyGenetics**
I've suggested to add the GitHub accounts of Translators to the repo.

Please let me know if you don't want your full name and affiliation included.

I want to acknowledge all contributions appropriately but we can remove personal information if you don't want it to be public.

@rkkmk @kozo2 @kyyonko @manabuishii @DaichiSaitoAF

### CODE REVIEW

File: Translators.md [hunk 1]
```diff
@@ -7,14 +7,50 @@ Team lead (co-ordinator)
 Members
 
 * [Riku Takei][takei_riku]: Assistant Research Fellow (University of Otago, Dunedin, New Zealand)
+  * Lesson organiser (Git)
```
Comment (joelnitta) [inline Translators.md]: Riku is also a reviewer, right?

Comment (TomKellyGenetics) [reply Translators.md]: yes

**Comment by TomKellyGenetics** [comment]
If everyone is okay with their details being listed, I will merge this. I think @kozo2 is the only one who has not agreed to this.

**Comment by TomKellyGenetics** [comment]
I've changed "reviewer" to moderator for the most part. I'd like to encourage peer-reviews between translators familiar with each lesson.

We also have some people interested in helping such as students. I'll add them as they get involved and I get their GitHub accounts. I'm happy to acknowledge anyone but at some point (if there are too many people to list), we may need to just leave it to commits apart from moderators and core translators.

**Pull request merged**